website/project/tasks.py

```diff
@@ -56,3 +57,57 @@ def on_node_updated(node_id, user_id, first_save, saved_fields, request_headers=
             }, headers={'Authorization': 'Bearer {}'.format(settings.SHARE_API_TOKEN), 'Content-Type': 'application/vnd.api+json'})
             logger.debug(resp.content)
             resp.raise_for_status()
+            if node.is_registration:
```

Comment:
This should be performed instead of the above request if the node is a registration.